packages/api-client/src/components/AddressBar/AddressBar.vue

```diff
@@ -315,26 +318,21 @@ function updateRequestPath(url: string) {
     opacity: 1;
   }
 }
-.codemirror-bg-switcher {
-  --scalar-background-1: color-mix(
-    in srgb,
-    var(--scalar-background-1),
-    var(--scalar-background-2)
-  );
-}
-.addressbar-bg-states:has(.cm-focused) .codemirror-bg-switcher {
-  --scalar-background-1: var(--scalar-background-1);
-}
 .addressbar-bg-states {
-  background: color-mix(
+  --addressbar-bg: color-mix(
     in srgb,
     var(--scalar-background-1),
     var(--scalar-background-2)
   );
+  background: var(--addressbar-bg);
 }
 .addressbar-bg-states:has(.cm-focused) {
-  background: var(--scalar-background-1);
+  --addressbar-bg: var(--scalar-background-1);
   border-color: var(--scalar-border-color);
   outline: 1px solid var(--scalar-color-accent);
 }
+.addressbar-bg-states:has(.cm-focused) .fade-left,
+.addressbar-bg-states:has(.cm-focused) .fade-right {
+  --addressbar-bg: var(--scalar-background-1);
```

Comment:
should we prefix this? or don’t we need this because it’s scoped to a CSS class?
but we should add a dash between adress and bar.

```suggestion
  --scalar-address-bar-bg: var(--scalar-background-1);
```